components/file-upload/src/index.js

```diff
@@ -40,10 +40,10 @@ const FileUpload = ({
   meta, children, hint, acceptedFormats, className,
 }) => (
   <Label className={className} error={meta.error}>
-    <LabelText error={meta.error}>{children}</LabelText>
+    <LabelText>{children}</LabelText>
```

Comment:
I'm not sure why LabelText has a prop 'error' in the first place, or why it would cause the label to be bold, if this isn't an established pattern.

Is this prop used anywhere else?

For reference: 

https://govuk-elements.herokuapp.com/form-elements/example-form-elements/

Reply:
Ah yes, I think it was added by mistake, it is also here: https://github.com/penx/govuk-react/blob/master/components/date-input/src/index.js#L77

It appears to make it bold due to this here: https://github.com/penx/govuk-react/blob/master/components/label-text/src/index.js#L33

I've noticed an un-required property on date-input also which I'll add to this branch.

Reply:
my point is that maybe https://github.com/penx/govuk-react/blob/master/components/label-text/src/index.js#L33 should be removed

Reply:
I agree and from what I can tell it can be removed as Labels should not render bold when there is an error (I cannot see any pattern or suggestion that they should be yet - perhaps this component was used to also render the error text at one point).

Reply:
sorry

Reply:
nonsense - thanks so much for all the work on this! Miss you, hope all is good!